libbeat/scripts/Makefile

```diff
@@ -8,6 +8,7 @@ export GO15VENDOREXPERIMENT=1
 GOFILES = $(shell find . -type f -name '*.go')
 SHELL=/bin/bash
 ES_HOST?="elasticsearch-210"
+BEAT_DIR?=github.com/elastic/beat
```

Comment:
It is `github.com/elastic/beats` instead of `beat` here, that is why the test are broken. Other LGTM.